services/openshift/scripts/openshift

```diff
@@ -113,15 +112,22 @@ if [ ! -f $master_config ]; then
     done
 
     echo "[INFO] Preparing OpenShift config"
-    start_openshift --write-config=${ORIGIN_DIR}/openshift.local.config > /dev/null 2>&1
+    start_openshift -d --write-config=${ORIGIN_DIR}/openshift.local.config > /dev/null 2>&1
     wait_for_config_files ${master_config} ${node_config}
 
+    echo "Create openshift-dev and admin User"
```

Comment:
That is something @hardy suggested to use. I was going with dev but he told me to use openshift-dev.